<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>general_fitting_options</class>
 <widget class="QWidget" name="general_fitting_options">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>20</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QCheckBox" name="simul_fit_checkbox">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Simultaneous over</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="simul_fit_by_combo"/>
   </item>
   <item>
    <widget class="QComboBox" name="simul_fit_by_specifier"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
